expyfun/_eyelink_controller.py

```diff
@@ -561,7 +562,17 @@ def custom_calibration(self, ctype='HV5', horiz=2./3., vert=2./3.,
                 raise ValueError('{0} too large ({1} > {2})'
                                  ''.format(s, p, m))
         # make the locations
-        mat = np.array([[0, 0], [1, 0], [-1, 0], [0, 1], [0, -1]])
+        if ctype == 'HV5':
+            mat = np.array([[0, 0], [1, 0], [-1, 0], [0, 1], [0, -1]])
+        elif ctype == 'HV9':
+            mat = np.array([[0, 0], [1, 0], [-1, 0], [0, 1], [0, -1], [1, 1],
+                            [-1, -1], [1, -1], [-1, 1]])
+        elif ctype == 'H3':
+            mat = np.array([[0, 0], [1, 0], [-1, 0]])
+        elif ctype == 'HV13':
+            mat = np.array([[0, 0], [1, 0], [-1, 0], [0, 1], [0, -1], [1, 1],
+                            [-1, -1], [1, -1], [-1, 1], [.5, .5], [-.5, -.5],
+                            [.5, -.5], [-.5, .5]])
```

Comment:
FYI these `mat` arrays will have different `dtype` (first four `int` last one `float`) because of their inputs which in principle could be dangerous, but it's probably okay :)